src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/CreateSessionTemplateRequestDto.kt

```diff
@@ -29,8 +29,6 @@ data class CreateSessionTemplateRequestDto(
   @Schema(description = "visit room", example = "A1", required = true)
   @field:NotBlank
   val visitRoom: String,
-  @Schema(description = "restrictions", required = false)
-  val restrictions: String? = null,
```

Comment:
Removed this because 'restrictions' is not used and we do not have data for it for this script.